Formula/mpich.rb

```diff
@@ -39,15 +48,26 @@ def install
     end
 
     system "./configure", "--disable-dependency-tracking",
+                          "--enable-fast=all,O3",
+                          "--enable-g=dbg",
+                          "--enable-romio",
+                          "--enable-shared",
+                          "--enable-sharedlibs=gcc-osx",
+                          "--with-pm=hydra",
+                          "CC=gcc-10",
+                          "CXX=g++-10",
+                          "FC=gfortran-10",
+                          "F77=gfortran-10",
```

Comment:
Are these not picked up automatically?

If not, I'd replace the `10` with `Formula["gcc"].any_installed_version.major`.

Reply:
Good point. I have pushed a commit addressing both issues.